tests/testthat/test-object_name_linter.R

```diff
@@ -151,3 +150,67 @@ test_that("object_name_linter won't fail if an imported namespace is unavailable
     3L
   )
 })
+
+test_that("object_name_linter supports custom regexes", {
```

Comment:
don't have any tests of the default-named `"custom"` case. I'm particularly interested how it looks when there's >1 default-named regex to check